test/uk/gov/hmrc/customs/emailfrontend/views/components/FieldsetSpec.scala

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.customs.emailfrontend.views.components
+
+import play.api.Application
+import org.scalatestplus.play._
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Document
+import play.api.Application
+import play.api.test._
+import play.api.test.Helpers._
+import play.api.inject.bind
+import play.twirl.api.{HtmlFormat, Html}
+import play.api.i18n.{
+  Messages,
+  MessagesApi,
+  MessagesImpl,
+  Lang,
+  DefaultMessagesApi
+}
+import uk.gov.hmrc.customs.emailfrontend.views.html.components.fieldset
+
+import uk.gov.hmrc.customs.emailfrontend.utils.{
```

Comment:
Could you please format like below?

import uk.gov.hmrc.customs.emailfrontend.utils.{FakeIdentifierAgentAction, SpecBase}